ui/src/main/webapp/src/app/reports/jpa/jpa-report.component.html

```diff
@@ -0,0 +1,157 @@
+<style>
+.panel-primary.wuTechnologies { border-color: #e7e7e7; }
+.panel-primary.wuTechnologies .panel-heading { background-color: white; padding: 15px 15px; border-bottom: 2px solid #e7e7e7; }
+.panel-primary.wuTechnologies .panel-heading .panel-title { color: black; font-size: 20px; font-weight: 500; }
```

Comment:
Are there any significant differences between styles of this table and other reports tables? Or could we use common styles here too?